app/assets/javascripts/views/shared/social-tooltip.js

```diff
@@ -0,0 +1,21 @@
+(function (App) {
+
+  App.View.SocialTooltip = App.Component.Tooltip.extend({
+
+    content: JST['templates/shared/social-tooltip'],
+
+    initialize: function (options) {
+      this.refElem = options.refElem;
+      this.constructor.__super__.initialize.call(this, options);
+    },
+    _showTooltip: function () {
```

Comment:
At the beginning I wasn't sure if I went to need anything else, so I prefered to use it  in a custom method just in case I needed to add anything else. It's not the case, so we can call its parent method directly.